# generated by patch-package 6.4.7 on 2021-07-12 13:20:50
#
# command:
#   npx patch-package @uniswap/v3-sdk
#
# declared package:
#   @uniswap/v3-sdk: 3.2.1
#
diff --git a/node_modules/@uniswap/v3-sdk/dist/v3-sdk.esm.js b/node_modules/@uniswap/v3-sdk/dist/v3-sdk.esm.js
index 3da216b..f271dde 100644
--- a/node_modules/@uniswap/v3-sdk/dist/v3-sdk.esm.js
+++ b/node_modules/@uniswap/v3-sdk/dist/v3-sdk.esm.js
@@ -4,9 +4,12 @@ import invariant from 'tiny-invariant';
 import { defaultAbiCoder, Interface } from '@ethersproject/abi';
 import { getCreate2Address } from '@ethersproject/address';
 import { keccak256, pack } from '@ethersproject/solidity';
-import { abi as abi$1 } from '@uniswap/v3-periphery/artifacts/contracts/NonfungiblePositionManager.sol/NonfungiblePositionManager.json';
-import { abi } from '@uniswap/v3-periphery/artifacts/contracts/interfaces/ISelfPermit.sol/ISelfPermit.json';
-import { abi as abi$2 } from '@uniswap/v3-periphery/artifacts/contracts/SwapRouter.sol/SwapRouter.json';
+import _json1 from '@uniswap/v3-periphery/artifacts/contracts/NonfungiblePositionManager.sol/NonfungiblePositionManager.json';
+import _json2 from '@uniswap/v3-periphery/artifacts/contracts/interfaces/ISelfPermit.sol/ISelfPermit.json';
+import _json3 from '@uniswap/v3-periphery/artifacts/contracts/SwapRouter.sol/SwapRouter.json';
+const abi$1 = _json1.abi
+const abi = _json2.abi
+const abi$2 = _json3.abi
 
 function asyncGeneratorStep(gen, resolve, reject, _next, _throw, key, arg) {
   try {
